ephios/user_management/migrations/0003_userprofile_calendar_token_squashed_0008_auto_20200925_1640.py

```diff
@@ -1,11 +1,12 @@
 # Generated by Django 3.1.1 on 2020-09-25 14:54
 
-from django.conf import settings
-from django.db import migrations, models
-import django.db.models.deletion
 import secrets
 import uuid
 
+import django.db.models.deletion
```

Comment:
do you apply isort locally? maybe you should add it to the repo ^^

Reply:
yes I have it on my list :)